sklearn/dummy.py

```diff
@@ -540,7 +540,7 @@ def __init__(self, *, strategy="mean", constant=None, quantile=None):
 
     @_fit_context(prefer_skip_nested_validation=True)
     def fit(self, X, y, sample_weight=None):
-        """Fit the random regressor.
+        """Fit the regressor.
```

Comment:
```suggestion
        """Fit the baseline regressor.
```

To ensure coherence with the [<code>DummyClassifier</code>](https://scikit-learn.org/stable/modules/generated/sklearn.dummy.DummyClassifier.html).